modules/accredited_representative_portal/lib/tasks/seed/staging_seed/methods.rb

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module StagingSeeds
+    RequestOptions = Struct.new(
+      :org, :rep, :claimant, :resolution_cycle,
+      :resolved_time, :unresolved_time, :totals,
+      keyword_init: true
+    )
+
+    module RequestMethods
+      def create_claimants(count = 10)
+        # just grab users to be claimants
+        UserAccount.limit(count).to_a
+      end
+
+      def create_request_with_resolution(options)
+        if rand < 0.5
+          created_at = options.resolved_time.next
+          request = create_poa_request(options.org, options.rep, options.claimant, created_at)
+          create_resolution(request, options.resolution_cycle.next)
+          options.totals[:resolutions] += 1
+        else
+          create_poa_request(options.org, options.rep, options.claimant, options.unresolved_time.next)
+        end
+
+        options.totals[:requests] += 1
+      end
+
+      def create_poa_request(org, rep, claimant, created_at)
+        request = PowerOfAttorneyRequest.new(
+          claimant_id: claimant.id,
+          claimant_type: 'veteran',
+          power_of_attorney_holder_type: org ? 'AccreditedOrganization' : 'AccreditedIndividual',
```

Comment:
FYI, I think we'll want to update these later to an official set, which didn't get properly codified in the code base up til now:
```
veteran_service_organization
claims_agent
attorney
```

OJ has an enum like this in his rep-data oriented changes recently. The enum should get centralized somewhere at some point.

cc @jvcAdHoc